meshroom/ui/qml/Viewer/CameraResponseGraph.qml

```diff
@@ -34,7 +35,8 @@ FloatingPane {
         onWheel: {}
     }
 
-    property bool crfReady: csvData.ready && csvData.nbColumns >= 4
+    // note: We need to use csvData.getNbColumns() slot instead of the csvData.nbColumns property to avoid a crash on linux.
+    property bool crfReady: csvData.ready && (csvData.getNbColumns() >= 4)
```

Comment:
To avoid a binding warning (I got one during testing), you could check if *csvData* exists in first place, like this:
```suggestion
    property bool crfReady: csvData && csvData.ready && (csvData.getNbColumns() >= 4)
```

Everything else seems to work fine.

Reply:
csvData is a static qml item, I do not see how it can be invalid.
```
CsvData {
        id: csvData
```

Reply:
I got this error:
```
TypeError: Cannot read property 'ready' of null
```